packages/windows_devices/lib/src/display/idisplaymonitor.dart

```diff
@@ -0,0 +1,540 @@
+    } finally {
+      free(retValuePtr);
+    }
+  }
+
+  Point get whitePoint {
+    final retValuePtr = calloc<NativePoint>();
+
+    try {
+      final hr = ptr.ref.vtable
+              .elementAt(21)
+              .cast<
+                  Pointer<
+                      NativeFunction<
+                          HRESULT Function(VTablePointer lpVtbl,
+                              Pointer<NativePoint> retValuePtr)>>>()
+              .value
+              .asFunction<
+                  int Function(VTablePointer lpVtbl,
+                      Pointer<NativePoint> retValuePtr)>()(
+          ptr.ref.lpVtbl, retValuePtr);
+
+      if (FAILED(hr)) throw WindowsException(hr);
+
+      return retValuePtr.toDart();
+    } finally {
+      free(retValuePtr);
+    }
+  }
+
+  double get maxLuminanceInNits {
+    final retValuePtr = calloc<Float>();
+
+    try {
+      final hr = ptr.ref.vtable
+          .elementAt(22)
+          .cast<
+              Pointer<
+                  NativeFunction<
+                      HRESULT Function(
+                          VTablePointer lpVtbl, Pointer<Float> retValuePtr)>>>()
+          .value
+          .asFunction<
+              int Function(VTablePointer lpVtbl,
+                  Pointer<Float> retValuePtr)>()(ptr.ref.lpVtbl, retValuePtr);
+
+      if (FAILED(hr)) throw WindowsException(hr);
+
+      return retValuePtr.value;
+    } finally {
+      free(retValuePtr);
+    }
+  }
+
+  double get minLuminanceInNits {
+    final retValuePtr = calloc<Float>();
+
+    try {
+      final hr = ptr.ref.vtable
+          .elementAt(23)
+          .cast<
+              Pointer<
+                  NativeFunction<
+                      HRESULT Function(
+                          VTablePointer lpVtbl, Pointer<Float> retValuePtr)>>>()
+          .value
+          .asFunction<
+              int Function(VTablePointer lpVtbl,
+                  Pointer<Float> retValuePtr)>()(ptr.ref.lpVtbl, retValuePtr);
+
+      if (FAILED(hr)) throw WindowsException(hr);
+
+      return retValuePtr.value;
+    } finally {
+      free(retValuePtr);
+    }
+  }
+
+  double get maxAverageFullFrameLuminanceInNits {
+    final retValuePtr = calloc<Float>();
+
+    try {
+      final hr = ptr.ref.vtable
+          .elementAt(24)
+          .cast<
+              Pointer<
+                  NativeFunction<
+                      HRESULT Function(
+                          VTablePointer lpVtbl, Pointer<Float> retValuePtr)>>>()
+          .value
+          .asFunction<
+              int Function(VTablePointer lpVtbl,
+                  Pointer<Float> retValuePtr)>()(ptr.ref.lpVtbl, retValuePtr);
+
+      if (FAILED(hr)) throw WindowsException(hr);
+
+      return retValuePtr.value;
+    } finally {
+      free(retValuePtr);
+    }
+  }
+
+  List<int> getDescriptor(DisplayMonitorDescriptorKind descriptorKind) {
+    final pValueSize = calloc<Uint32>();
+
+    // @halildurmus NOTE: I had to manually change this from calloc<Uint8>
```

Comment:
@halildurmus Flagging here -- this calloc was generated as:

```dart
    final retValuePtr = calloc<Uint8>();
```

but should be as shown below.

Reply:
Never mind, I just fixed it. It was an easy one.